website/project/model.py

```diff
@@ -3681,6 +3683,10 @@ def institutions_url(self):
     def institutions_relationship_url(self):
         return self.absolute_api_v2_url + 'relationships/institutions/'
 
+    def flip_notification_settings_dirty(self):
+        self.notification_settings_dirty = True
+        self.save()
```

Comment:
We try to avoid implicit saves as much as possible because they may cause unwanted side effects--especially risky given how much logic is in `Node#save`.

```python
node.flip_notification_settings.dirty()  # implicit save
node.category = 'data'
node.save()   # another save
```

I think we can get ride of this method; setter methods in Python are usually unnecessary.

```python
node.notification_settings_dirty = True
node.category = 'data'
node.save()
```